components/client.jsx

```diff
@@ -271,6 +273,18 @@ export default class Client extends Component{
         });
     }
 
+    /** Switch room */
+    switchRoom(args) {
+        if (typeof args != 'string' && !this.isGuest) return;
```

Comment:
why is this conditional on not being a guest?

Reply:
Apparently when signed as a guest, `getRoom()` doesn't work.  It might work if I make them join the room but I wasn't unsure if I only meant to leave them in read-only mode (without joining). So for now, I left guest mode out of this.